<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="previewFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>901</width>
         <height>766</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="PsView" name="preview">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionPrintAndClose"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreferences"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="layoutBox">
       <property name="title">
        <string>Layout</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="LayoutRadioButton" name="layout1UpBtn">
          <property name="text">
           <string>1 up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LayoutRadioButton" name="layout2UpBtn">
          <property name="text">
           <string>2 up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LayoutRadioButton" name="layout4UpBtn">
          <property name="text">
           <string>4 up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LayoutRadioButton" name="layout8UpBtn">
          <property name="text">
           <string>8 up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LayoutRadioButton" name="layoutBookletBtn">
          <property name="text">
           <string>Booklet</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="filesBox">
       <property name="title">
        <string>Jobs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="PsFilesListView" name="filesView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="printerBox">
       <property name="title">
        <string>Printer</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="PrintersComboBox" name="printersCbx"/>
        </item>
        <item>
         <widget class="QToolButton" name="printerConfigBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Configure ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrintAndClose"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionPreviousSheet"/>
   <addaction name="actionNextSheet"/>
  </widget>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPrintAndClose">
   <property name="text">
    <string>Print and Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNextSheet">
   <property name="text">
    <string>Next sheet</string>
   </property>
  </action>
  <action name="actionPreviousSheet">
   <property name="text">
    <string>Previous sheet</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayoutRadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">widgets.h</header>
  </customwidget>
  <customwidget>
   <class>PrintersComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets.h</header>
  </customwidget>
  <customwidget>
   <class>PsFilesListView</class>
   <extends>QListWidget</extends>
   <header location="global">widgets.h</header>
  </customwidget>
  <customwidget>
   <class>PsView</class>
   <extends>QFrame</extends>
   <header>psview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
